# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/websocket/BaseWebSocketChannel.h
# Commit: 712cd584e42f
# Full Hash: 712cd584e42f0be70b18a38cbf492450af2c503f
# Author: Randell Jesup <rjesup@wgate.com>
# Date: 2022-02-12 09:47:43
# Regressor Bug: 1749786
# File Overlap Count: 1
# Description:
#   Bug 1749786 - websocketchannel cleanup r=necko-reviewers,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D135727
# ==============================================================================

diff -r 872873e809a6 -r 712cd584e42f netwerk/protocol/websocket/BaseWebSocketChannel.h
--- a/netwerk/protocol/websocket/BaseWebSocketChannel.h	Fri Feb 11 17:12:45 2022 +0000
+++ b/netwerk/protocol/websocket/BaseWebSocketChannel.h	Fri Feb 11 17:14:07 2022 +0000
@@ -103,7 +103,8 @@
   nsCOMPtr<nsILoadInfo> mLoadInfo;
   nsCOMPtr<nsITransportProvider> mServerTransportProvider;
 
-  // Used to ensure atomicity of mTargetThread
+  // Used to ensure atomicity of mTargetThread.
+  // Set before AsyncOpen via RetargetDeliveryTo or in AsyncOpen, never changed after AsyncOpen
   DataMutex<nsCOMPtr<nsIEventTarget>> mTargetThread{
       "BaseWebSocketChannel::EventTargetMutex"};
 